examples/gltf/app.js

```diff
@@ -50,12 +50,11 @@ function loadGLTF(urlOrPromise, gl, options = DEFAULT_OPTIONS) {
     gltfParser = new GLTFParser({uri: urlOrPromise});
     return gltfParser.parseAsync(data);
   }).then(gltf => {
-    const instantiator = new GLTFInstantiator(gl, options);
-    const scenes = instantiator.instantiate(gltf);
-    const animator = instantiator.createAnimator();
+
+    const {scenes, animator} = createGLTFObjects(gl, gltf)
```

Comment:
What happened to options??

Reply:
> What happened to options??

Thanks, that was it! That fixed the iffy rendering in the example!